cmd/storage-rest-client.go

```diff
@@ -147,6 +147,17 @@ func (client *storageRESTClient) IsOnline() bool {
 	return atomic.LoadInt32(&client.connected) == 1
 }
 
+func (client *storageRESTClient) CrawlAndGetDataUsage() (DataUsageInfo, error) {
+	respBody, err := client.call(storageRESTMethodCrawlAndGetDataUsage, nil, nil, -1)
+	if err != nil {
+		return DataUsageInfo{}, err
+	}
+	defer http.DrainBody(respBody)
+	var usageInfo DataUsageInfo
+	err = gob.NewDecoder(respBody).Decode(&usageInfo)
```

Comment:
Note that you have to watch for space characters have a look at `func (client *storageRESTClient) VerifyFile()`

Reply:
> Note that you have to watch for space characters, have a look at `func (client *storageRESTClient) VerifyFile()`

ops yes, I thought it is like xml or json where it is okay to have some leading spaces.

But it is fixed now